.travis.yml

```diff
@@ -32,8 +32,9 @@ install:
   #- conda config --add channels mantid
   #- conda install --yes -c mantid/label/nightly mantid-framework
   - pip install --upgrade codecov
-  - pip install flake8
-  
+  - pip install -r requirements.txt
```

Comment:
You are going to use `pip` but not `Conda`? 

And how do you plan to make `mantid` work?

Reply:
Enabling the CI/CD is not in the scope of this task. I only included these lines so that Michael and Donnie are aware of the requirements when they enable the CI/CD